modules/context/repo.go

```diff
@@ -441,6 +441,7 @@ func RepoAssignment(ctx *Context) (cancel context.CancelFunc) {
 	ctx.Repo.Owner = owner
 	ctx.ContextUser = owner
 	ctx.Data["Username"] = ctx.Repo.Owner.Name
+	ctx.ContextUser = owner
```

Comment:
```suggestion
	ctx.ContextUser = owner
	ctx.Data["Username"] = ctx.Repo.Owner.Name
```

Duplicated `	ctx.ContextUser = owner`